packages/core/entity/FileDetail/index.ts

```diff
@@ -180,4 +181,17 @@ export default class FileDetail {
         }
         return this.thumbnail;
     }
+
+    public getLinkToPlateUI(baseURL: string): string | undefined {
+        // Grabbing plate barcode
+        const platebarcode = this.getFirstAnnotationValue(AnnotationName.PLATE_BARCODE);
+
+        if (!platebarcode) {
+            return undefined;
+        }
+
+        // LabKey does not support HTTPS yet
+        const baseURLHttp = baseURL.replace("https", "http");
+        return `${baseURLHttp}/labkey/aics_microscopy/AICS/editPlate.view?Barcode=${platebarcode}}`;
```

Comment:
This still seems to redirect to http for me?

Reply:
lol that is a copy/paste issue seems we went with `https` at some point - will remove

Reply:
Removed